test/test_point_inclusion.cpp

```diff
@@ -360,8 +373,10 @@ TEST(CylinderPointContainment, Basic)
 
     // near two-axis maximum
     const double sq2 = sqrt(2) / 2;
```

Comment:
Looks like sq2 is only used in a single location. Do you mind to use sq2_eps for this location too, thus allowing to rename s2_eps back to sq2. This way, the bugfix wouldn't require too many changes for the test.

Reply:
Yeah, I don't like the name very much. But it don't have a better idea either. I'm not so strong on names ;-)

Reply:
I renamed `sq2_eps` to `sq2e`. Also, to keep consistency throughout the code, I renamed `sq2` to `sq2e` in the sphere test.